init-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,21 @@
+spring:
+  profiles: default
+  cloud:
+    config:
+      # If cannot connect to config server, don't start the application
+      fail-fast: true
+      discovery:
+        enabled: true
+        service-id: CONFIG-SERVER
+      retry:
+        initialInterval: 2000
+        multiplier: 1.5
+        maxInterval: 60000
+        maxAttempts: 100
+      label: main
```

Comment:
I think @eladiw is using a parameter for this value since it's not always main

Reply:
i think this is redundant. why we need a label here? if we need then either remove it from here or leave as a placeholder with an invalid value.

Reply:
Removed from everywhere